src/ApiService/index.js

```diff
@@ -0,0 +1,6 @@
+import { addCart } from "./Cart/addCart";
+import { getCart } from "./Cart/getCart";
+import { removeCart } from "./Cart/removeCart";
+import { addWishlist } from "./Wishlist/addWishlist";
+import { removeWishlist } from "./Wishlist/removeWishlist";
```

Comment:
can try direct export

Reply:
thanks to point out this one. changes will see in next commit